en/docs/how-tos/endpoints/overriding-endpoint-information/overriding-endpoints-for-imported-apis.md

```diff
@@ -3,237 +3,115 @@
-            - sand - Use this for a sandbox endpoint.
-        - <endpoint-index> - The endpoint index starts from 0. If there are many URLs (e.g., load-balanced or failover) , you can override them using endpoint indexes (e.g., 1, 2, 3) .
-        - <path-to-MGW-executable-file> - Enter the path to the WSO2 API Microgateway project's executable file ( `.jar` ).
-       ```
+    ``` text tab="Override Multiple endpoints"
+    gateway <path-to-MGW-jar> --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint-URL-1>" --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint_URL_2>" --<API-ID>_<endpoint-type>_basic_username="<basic-auth-username>" --<API-ID>_basic_password="<basic-auth-password>
+    ```
+   
+    - `<basic-auth-username>` - Provide your username                            
+    - `<basic-auth-password>` - Provide your password                               
+   
+    ``` text tab="Example 1 - single endpoint"
+    gateway /home/user/petstore-project/target/petstore-project.jar  
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com"
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin" 
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_basic_password="admin"
+    ```
     
-       ``` java tab="Example"
-         gateway /home/user/petstore-project/target/petstore-project.jar --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com" --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_1="http://support.wso2.com"
-       ```
+    ``` text tab="Example 2 - multiple endpoints"
+    gateway /home/user/petstore-project/target/petstore-project.jar 
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_0="http://wso2.com" 
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_endpoint_1="http://support.wso2.com"
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_prod_basic_username="admin" 
+    --60f7111f-fdc5-4cc7-b497-1cea64c6a97f_basic_password="admin"
+    ```
+ 
+    !!! Note
+        If you do not specify a username, the username that you defined in the API definition, which you specified during the API implementation, is used.
 
 
-#### Overriding multiple endpoints at runtime which are protected by basic authentication
+#### Overriding endpoints using system properties
 
-Use the following command to provide the system variables at server start-up to override more than one endpoint (e.g., load balance endpoint) that corresponds to an API, which was imported from WSO2 API Manager.
+Use the following command to provide the system variables as system properties to override more than one endpoint (e.g., load balance endpoint) that corresponds to an API, which was imported from WSO2 API Manager.
 
-Follow the instructions below to override multiple endpoints, which are either a production or sandbox endpoint:
+Follow the instructions below to override single endpoint or multiple endpoints, which are either production or sandbox endpoints.
```

Comment:
Follow the instructions below to override the production or sandbox endpoint(s).